### PR TITLE
tickets/DM-33687: Deprecated this technote.

### PR DESCRIPTION
**Pull request by rbovill**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -1,34 +0,0 @@
...
-      - name: Build
-        run: |
-          make html
-      - name: Upload
-        if: ${{ github.event_name == 'push' }}
-        env:
-          LTD_PASSWORD: ${{ secrets.LTD_PASSWORD }}
-          LTD_USERNAME: ${{ secrets.LTD_USERNAME }}
-        run: |
-          ltd upload --gh --dir _build/html --product tstn-007
```
Comment (couger01) [inline .github/workflows/ci.yaml]: This file should not be deleted.

File: conf.py [hunk 2]
```diff
@@ -1,22 +0,0 @@
...
-with open(metadata_path, 'r') as f:
-    confs = configure_technote(f)
-g = globals()
-g.update(confs)
-
-# Add intersphinx inventories as needed
-# http://www.sphinx-doc.org/en/stable/ext/intersphinx.html
-# Example:
-#
-#     intersphinx_mapping['python'] = ('https://docs.python.org/3', None)
```
Comment (couger01) [inline conf.py]: This file should not be deleted.

File: index.rst [hunk 3]
```diff
@@ -42,26 +42,7 @@
...
-This tech note describes the infrastructure and process for building and releasins T&S software.  It should be considered the continuation of the `TSSW Developer Guide
-<https://tssw-developer.lsst.io/>`_.
-
-.. Add content here.
-.. Do not include the document title (it's automatically added from metadata.yaml).
-
-.. .. rubric:: References
-
-.. Make in-text citations with: :cite:`bibkey`.
+   **This technote was moved to Observatory Operations.**
```
Comment (couger01) [inline index.rst:47]: Maybe a link to the relevant page.

File: requirements.txt [hunk 4]
```diff
@@ -1 +0,0 @@
-documenteer[technote]>=0.5.4,<0.6.0
```
Comment (couger01) [inline requirements.txt]: This file should not be removed.

**Review by couger01 — CHANGES_REQUESTED** [review]
Need to keep the build infrastructure otherwise it won't build and deploy to LSST the Docs.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (rbovill) [reply .github/workflows/ci.yaml]: Restored.

File: conf.py (hunk 2 above)
Comment (rbovill) [reply conf.py]: Restored.

File: index.rst (hunk 3 above)
Comment (rbovill) [reply index.rst:47]: Link added.

File: requirements.txt (hunk 4 above)
Comment (rbovill) [reply requirements.txt]: Restored.

**Review by couger01 — APPROVED** [review]
LGTM

**Pull request merged**